tests/conftest.py

```diff
@@ -10,6 +10,16 @@
 
 from qibo.backends import _Global, construct_backend
 
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--gpu_only",
```

Comment:
```suggestion
        "--gpu-only",
```

Sorry, it was my fault...